ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -109,6 +111,9 @@ function mhrStaffPaymentParameters (staffPayment: StaffPaymentIF) {
     if (staffPayment.isPriority) {
       paymentParams = paymentParams + '&priority=true'
     }
+    if (staffPayment.isCreditCard) {
```

Comment:
Should this be ccPayment?

Reply:
You are correct! copypasta error in the param itself. 

On having another look, i need to refactor into the switch itself, as this will be a explicit payment option, not a checkbox(ie in addition to payment)